openshift/system/config/message_bus.yml

```diff
@@ -4,6 +4,7 @@ base: &default
     url: <%= ENV.fetch('MESSAGE_BUS_REDIS_URL', 'redis://system-redis/8') %>
     pool_size: <%= ENV.fetch('RAILS_MAX_THREADS', 5) %>
     pool_timeout: 5
+    namespace: system-mbus
```

Comment:
I think we should not do that
If people want to use Redis Enterprise it is an extra configuration and we should document it
Not making everyone migrate their data to namespaced redis

Reply:
Removing this would mean more changes in https://github.com/3scale/3scale-operator/pull/114.

Reply:
OK then this must be changes then. I will comment there